### DISCUSSION TITLE
'Uncaught TypeError: can't access property "get", this.$wire is undefined' when first installed

### DISCUSSION THREAD

**Discussion by rama-can** [Power-Components/livewire-powergrid#1229] [Q&A]
**hello how to solve this?**
this happens because of `Filter::datepicker('created_at'),`
```
Uncaught TypeError: can't access property "get", this.$wire is undefined
    init powergrid.js:2
    generateEvaluatorFromFunction module.esm.js:573
    tryCatch module.esm.js:527
    evaluate module.esm.js:553
    <anonymous> module.esm.js:2928
    flushHandlers module.esm.js:698
    stopDeferring module.esm.js:703
    deferHandlingDirectives module.esm.js:706
    initTree module.esm.js:186
    start module.esm.js:141
    start module.esm.js:140
    <anonymous> app.js:10
powergrid.js:2:15333
```
**I have followed the installation steps in the documentation**
### This is my first time installing PowerGrid 5.1 on: 
**laravel** 
10.10
**livewire** 
3
**alpine** 
3.x
**tailwind**
3.x

**Comment by akulmehta** [comment]
I am getting the same error but with a different stack of reports and was wondering if you ever resolved it. I have also created an issue regarding this: https://github.com/Power-Components/livewire-powergrid/issues/1405
